### ISSUE TITLE
resolveLatestVersion is not compatible with GitHub Enterprise Server

### ISSUE THREAD

**Issue by Sidewinder53** [Azure/use-kubelogin#3]
When running `azure/use-kubelogin@v1` on GitHub Enterprise Server (GHES) specifying `latest` for `kubelogin-version` will attempt to fetch `Azure/kubelogin` from the local GHES instance:

https://github.com/Azure/use-kubelogin/blob/f2ad8f361a76ba2820e5b2bdbdccf2fe6bf85233/src/gh.ts#L11-L24

This will result in errors like this:
```
RequestError [HttpError]: invalid json response body at https://<GHES endpoint>/api/v3/repos/Azure/kubelogin/releases/latest reason: Unexpected end of JSON input
```

resolveLatestVersion should either fetch kubelogin from a configurable endpoint or always from GitHub.com, but never trough the default Octokit endpoint.

**Comment by bcho** [comment]
Thanks for reporting this. I am not very familiar with GHES, @Sidewinder53 can you confirm inside the GHES environment, github.com is accessible?

**Comment by Sidewinder53** [comment]
Yes, github.com is accessible from the GHES environment. When specifying `kubelogin-version: v0.0.31` in the action inputs, the kubelogin release will be fetched from github.com just fine. 

My understanding is, that `resolveLatestVersion` uses Octokit to resolve the latest kubelogin release from the GitHub API. Octokit will, by default, use the local GitHub API for any requests. In the case of GHES, this is the local GHES instance instead of GitHub.com.

**Comment by bcho** [comment]
Hi @Sidewinder53 when you have a chance, could you run verify my fix in #4? (Sorry I don't have a GHES environment can test at this moment) We can use this version: `azure/use-kubelogin@45e09682adb5a0cad41a04ddd56a806dfeb79b4a`, example:

https://github.com/bcho/use-kubelogin-demo/pull/1/files

**Comment by Sidewinder53** [comment]
Confirmed to be working on GHES. Thanks @bcho for delivering this fix so fast, I appreciate it! 👍

**Comment by bcho** [comment]
thanks for confirming, I have published a new version `v1.1` with this fix.
